optimize_epanet.py

```diff
@@ -0,0 +1,126 @@
+from scipy.optimize import differential_evolution
+import subprocess
+import shutil
+from file_parser.parse_output_file import get_simulation_results
+from calculate_error import calculate_error
+import sys
+
+junctions_to_adjust = [" SO/ZD ", " SW/SO ", " SW/RO ", " SD/GO1 ", " SW/AN ", " HP7 "]
+pipes_to_adjust = [" p23", " p129", " p159"]
+
+
+def update_epanet_file(filename, x):
+    # read EPANET file
+    with open(filename) as model_file:
+        lines = model_file.readlines()
+
+    junctions_index = lines.index("[JUNCTIONS]\n") + 2
+    pipes_index = lines.index("[PIPES]\n") + 2
+
+    # update junctions
+    for i in range(len(junctions_to_adjust)):
```

Comment:
Wsm to nie wiem czy możemy demand optymalizować?

Reply:
Bo ja tam pamiętam, że jakąś głupotę wpisałam. Możemy np. teraz zostawić tylko x dla pipes (Roughness)